src/components/DialogContentContainer/DialogContentContainer.tsx

```diff
@@ -1,12 +1,21 @@
 import { SIZES } from "../../constants/sizes";
 import React, { useRef, forwardRef } from "react";
-import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { BEMClass } from "../../helpers/bem-helper";
-
+import VibeComponentProps from "src/types/VibeComponentProps";
 import "./DialogContentContainer.scss";
 
+interface DialogContentContainerProps extends VibeComponentProps {
+  children: React.ReactNode;
+  className: string;
+  ariaLabelledby?: string;
+  ariaDescribedby?: string;
+  type: "modal" | "popover";
+  size: "small" | "medium" | "large";
+  style: React.CSSProperties;
+}
+
 const DIALOG_TYPES = {
```

Comment:
Please create new file which called DialogContentContainerConstants.ts which will contain the enums of the DialogContentContainer component.
Please create an enum for the following: DIALOG_TYPES => DialogType, DIALOG_SIZES => DialogSize.
And replace the values of types and sizes with the following.
Also, use these enums for the type and size default props